src/fasttext.cc

```diff
@@ -430,7 +458,7 @@ void FastText::predict(
       }
       std::cout << dict_->getLabel(it->second);
       if (print_prob) {
-        std::cout << " " << std::exp(it->first);
+        std::cout << separator << std::exp(it->first);
```

Comment:
It looks like you've changed the default separator from a space to a tab – so this would be a breaking change. Might be better to keep the default as a space?

Reply:
leaving it along, it kept the default value: space